docs/api/modal.md

```diff
@@ -124,6 +142,58 @@ Setting a callback function should be used when you have complex dismissing crit
 
 Note that setting a callback function will cause the swipe gesture to be interrupted when using a card or sheet modal. This is because Ionic does not know what your callback function will resolve to ahead of time.
 
+TODO: Playground Example
+
+## Types of modals
+
+### Full screen
+
+#### Basic
+
+TODO: Playground Example
+
+### Card Modal
+
+Developers can create a card modal effect where the modal appears as a card stacked on top of your app's main content. To create a card modal, developers need to set the `presentingElement` property and the `swipeToClose` properties on `ion-modal`.
+
+The `presentingElement` property accepts a reference to the element that should display under your modal. This is typically a reference to `ion-router-outlet`.
+
+The `swipeToClose` property can be used to control whether or not the card modal can be swiped to close.
+
+#### Basic
+
+TODO: Playground Example
+
+#### Custom sizes
+
+TODO: Playground Example
+
+### Sheet Modal
+
+Developers can create a sheet modal effect similar to the drawer components available in maps applications. To create a sheet modal, developers need to set the `breakpoints` and `initialBreakpoint` properties on `ion-modal`.
+
+The `breakpoints` property accepts an array which states each breakpoint that the sheet can snap to when swiped. A `breakpoints` property of `[0, 0.5, 1]` would indicate that the sheet can be swiped to show 0% of the modal, 50% of the modal, and 100% of the modal. When the modal is swiped to 0%, the modal will be automatically dismissed.
+
+The `initialBreakpoint` property is required so that the sheet modal knows which breakpoint to start at when presenting. The `initialBreakpoint` value must also exist in the `breakpoints` array. Given a `breakpoints` value of `[0, 0.5, 1]`, an `initialBreakpoint` value of `0.5` would be valid as `0.5` is in the `breakpoints` array. An `initialBreakpoint` value of `0.25` would not be valid as `0.25` does not exist in the `breakpoints` array.
+
+The `backdropBreakpoint` property can be used to customize the point at which the `ion-backdrop` will begin to fade in. This is useful when creating interfaces that have content underneath the sheet that should remain interactive. A common use case is a sheet modal that overlays a map where the map is interactive until the sheet is fully expanded.
+
+:::note
+ Note: The `swipeToClose` property has no effect when using a sheet modal as sheet modals must be swipeable in order to be usable.
+:::
+
+#### Using breakpoints
+
+TODO: Playground Example
+
+#### Interacting with background content
+
+TODO: Playground Example
+
+#### Moving to a breakpoint
+
+TODO: Playground Example
+
 ## Interfaces
```

Comment:
This isn't strictly relevant, but it's exacerbated by the removal of the Usage section -- I noticed that there are two Interfaces sections, with slightly different content. 🤔 We should probably consolidate them here.